dbcrossbarlib/src/drivers/postgres_shared/column.rs

```diff
@@ -0,0 +1,81 @@
+//! PostgreSQL columns.
+
+use std::fmt;
+
+use super::{PgDataType, PgScalarDataType};
+use crate::common::*;
+use crate::schema::Column;
+
+/// A PostgreSQL identifier. This will be printed with quotes as necessary to
+/// prevent clashes with keywords.
+pub(crate) struct Ident<'a>(pub(crate) &'a str);
+
+impl<'a> fmt::Display for Ident<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // TODO: For now, we just quote everything, and not even necessarily
+        // correctly.
+        write!(f, "{:?}", self.0)
```

Comment:
so is this basically ruby's `String#inspect` which will give you double-quotes etc.?

of all the things, you gotta figure there's a rust lib that already does postgres value and ident quoting

Reply:
Yup, I pretty sure there's a library somewhere for that. I'll also need one for BigQuery at some point. For now, since we're just dealing with column identifiers and they're under our control. So I'm cheating, so that we can _use_ this ASAP. :-) This may break if your database columns are named `a\\"b`, or something like that, which I decided was OK for version 1, as long as the escaping code was 100% centralized so we could replace it later.